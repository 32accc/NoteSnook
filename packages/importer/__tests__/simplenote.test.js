const simplenote = require("../providers/simplenote");
const utils = require("./utils");

test("convert simplenote backup zip file", () => {
  let file = utils.getFile("simplenote.zip");

  expect(simplenote.convert([file])).toMatchSnapshot("simplenote-backup");
});