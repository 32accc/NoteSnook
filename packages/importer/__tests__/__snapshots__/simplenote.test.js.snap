// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert simplenote backup zip file: simplenote-backup 1`] = `
Array [
  Object {
    "content": Object {
      "data": "<ol>
<li>We Create</li>
<li>We Contribute</li>
<li>We Blog</li>
<li>We Help</li>
<li>We Love Coding</li>
<li>Work with us</li>
<li>Our Team</li>
<li>What we can do</li>
<li>Careers</li>
<li>Blog Page</li>
</ol>",
      "type": "html",
    },
    "dateCreated": "2019-09-25T19:12:04.794Z",
    "dateEdited": undefined,
    "pinned": true,
    "title": "KEYWORDS LIST OF MINT PLAYER",
  },
  Object {
    "content": Object {
      "data": "<ol>
<li>We Create</li>
<li>We Contribute</li>
<li>We Blog</li>
<li>We Help</li>
<li>We Love Coding</li>
<li>Work with us</li>
<li>Our Team</li>
<li>What we can do</li>
<li>Careers</li>
<li>Blog Page</li>
</ol>",
      "type": "html",
    },
    "dateCreated": "2019-09-21T17:43:30.648Z",
    "dateEdited": undefined,
    "tags": Array [
      "cow",
      "hello",
    ],
    "title": "List of Pages",
  },
]
`;
